OHHTTPStubs/Sources/NSURLSession/OHHTTPStubsMethodSwizzling.h

```diff
@@ -0,0 +1,51 @@
+/***********************************************************************************
+ *
+ * Copyright (c) 2012 Olivier Halligon, 2016 Sebastian Hagedorn
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ ***********************************************************************************/
+
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
This header file should not be part of the public API. You might want to alter the `podspec` accordingly [here](https://github.com/AliSoftware/OHHTTPStubs/blob/master/OHHTTPStubs.podspec#L55) to make it private (and thus make sure consumers of the pod won't be able to `#import` it from their app project)